includes/service-providers/constant_contact/class-newspack-newsletters-constant-contact-sdk.php

```diff
@@ -622,4 +625,138 @@ public function upsert_contact( $email_address, $data = [] ) {
 		);
 		return $this->get_contact( $email_address );
 	}
+
+	/**
+	 * Fetches a tag by its name
+	 *
+	 * Constant Contact API does not offer an endpoint to search for a tag by its name, so we have to query for all tags and look through them.
+	 *
+	 * If there are too many tags, we might not go over them all might fail in finding it, but that is a known limitation for now.
+	 * That's why we chose to do a case insensitive search, to increase the chances of finding the tag.
+	 *
+	 * @param string $name The tag name you are looking for. Case insensitive.
+	 * @return stdClass|WP_Error The tag object or a WP_Error if the tag was not found.
+	 */
+	public function get_tag_by_name( $name ) {
+		$limit_attempts     = 10;
+		$itemps_per_attempt = 100;
```

Comment:
```suggestion
		$items_per_attempt = 100;
```

Typo?

Reply:
fixed in https://github.com/Automattic/newspack-newsletters/pull/1054/commits/d71973c7cf98c46e271efb617dee2b0ab977a681